<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>551</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Usuarios Activos</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>561</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Formulario de Usuario.</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_apellido_nombre_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>25</y>
           <width>90</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Apellido y Nombre:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>40</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>D.N.I.:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_dni_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>45</y>
           <width>90</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>50</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Domicilio:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_domicilio_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>67</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>65</x>
           <y>175</y>
           <width>30</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_email_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>170</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>200</y>
           <width>40</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Usuario:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_usuario_acceso">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>200</y>
           <width>250</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>50</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_password_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>230</y>
           <width>190</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>260</y>
           <width>20</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Rol:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_rol_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>255</y>
           <width>191</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>admin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>editor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>visor</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_foto_usuario">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>30</y>
           <width>104</width>
           <height>104</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>104</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 50px;
min-width: 100px;
min-height: 100px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="btnFoto">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>280</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_ruta_foto">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>305</y>
           <width>450</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>285</y>
           <width>30</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Foto:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnAgregarUsuario">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>330</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnModificarUsuario">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>330</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Modificar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnEliminarUsuario">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>330</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Desactivar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnLimpiarFormulario">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>330</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tabla_usuarios_activos">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>360</y>
           <width>520</width>
           <height>150</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>id Usuario</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>D.N.I</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Domicilio</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Localidad</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Provincia</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Teléfono</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Email</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Usuario</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Contraseña</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ruta Foto</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rol</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>95</y>
           <width>50</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Localidad:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_localidad_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>90</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Seleccione...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25 de Mayo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9 de Julio</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Albardon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Angaco</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Calingasta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Capital</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Caucete</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chimbas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Iglesia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jachal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pocito</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rawson</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rivadavia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>San Martin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Santa Lucia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sarmiento</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ullum</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Valle Fertil</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Zonda</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>120</y>
           <width>50</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Provincia:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_provincia_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>115</y>
           <width>201</width>
           <height>22</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Seleccione...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Buenos Aires</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CABA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Catamarca</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Córdoba</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Corrientes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chaco</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chubut</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Entre Ríos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Formosa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jujuy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>La Pampa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>La Rioja</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mendoza</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Misiones</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neuquen</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Río Negro</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Salta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>San Juan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>San Luis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Santa Cruz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Santa Fe</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Santiago del Estero</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tierra del Fuego. Antartida e Islas del Sur</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tucumán</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>150</y>
           <width>50</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Teléfono:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_telefono_usuario">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>145</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnCerrar">
         <property name="geometry">
          <rect>
           <x>494</x>
           <y>10</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border : none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/iconos/cerrar.png</normaloff>../assets/iconos/cerrar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Usuarios Inactivos</string>
    </attribute>
    <widget class="QTableWidget" name="tabla_usuarios_inactivos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>501</width>
       <height>371</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>id Usuario</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D.N.I</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Domicilio</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Localidad</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Provincia</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contraseña</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ruta Foto</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rol</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnReactivarUsuario">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>110</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Reactivar usuario</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
